### DISCUSSION TITLE
Is the plugin is ready yet?

### DISCUSSION THREAD

**Discussion by radiaku** [cefsharp/CefSharp.Dom#4] [Q&A] [answered]
Hello @amaitland  , I Already built the plugin and try it as example. but GetPuppeteerPageAsync or .GetContentAsync(); is not found. 

Try to search it on this repository there is no function called like that, nor on pupettersharp.

Did I do it wrong on here?

**Comment by amaitland** [comment, marked as answer]
The project is available on Nuget.org at https://www.nuget.org/packages/CefSharp.Puppeteer/

Make sure you add a using statement for CefSharp.Puppeteer namespace.

> but GetPuppeteerPageAsync 

https://github.com/cefsharp/Puppeteer/blob/main/lib/CefSharp.Puppeteer/PuppeteerExtensions.cs#L16

The project is still in it's infancy, your welcome to contribute or sponsor my efforts to ensure the project evolves. 

https://github.com/sponsors/amaitland

**Reply by radiaku** [reply]
Thanks for replying.. But I found something weird
![Screenshot_271](https://user-images.githubusercontent.com/5983350/148121560-ec7f2e8c-95a4-4c02-be13-dbe6385e1814.png)

if I searching in this repository, not found.

Btw thanks for the nugget version is helpful.

> The project is still in it's infancy, your welcome to contribute or sponsor my efforts to ensure the project evolves.

I will.

**Reply by amaitland** [reply]
> if I searching in this repository, not found.

If you click the code tab you'll see an error message from GitHub saying `Sorry, forked repositories are not currently searchable.` So the behaviour you are seeing is expected.

**Reply by radiaku** [reply]
Ohh I see,  thank for explanation.

Btw I found error on this plugin, only working on x64 not on x82

![Screenshot_272](https://user-images.githubusercontent.com/5983350/148169582-e4816b31-40cb-436e-a5dd-6f40ff447d14.png)

Error code
`There was a mismatch between the processor architecture of the project being built "x86" and the processor architecture of the reference "CefSharp.Puppeteer, Version=1.0.22.0, Culture=neutral, PublicKeyToken=40c4b6fc221f4138, processorArchitecture=AMD64", "AMD64". This mismatch may cause runtime failures. Please consider changing the targeted processor architecture of your project through the Configuration Manager so as to align the processor architectures between your project and references, or take a dependency on references with a processor architecture that matches the targeted processor architecture of your project.	`

Or x82 is not supported?

Edit:
I tried build my self with x82 but not working.

**Reply by amaitland** [reply]
> Or x82 is not supported?

If you mean x86 then it's probably a mistake in the csproj file, something that can likely be fixed.

**Reply by radiaku** [reply]
Yes, I mean x86. When installing from nuget and my project architecture is x86, its failed. Now I'm still reverting to socket connect cefsharp until next cefsharp.Puppeteer release.
